### PR TITLE
Use native links for public archive cards

### PR DESCRIPTION
**Pull request by MLGalusha**
Archive cards, collection highlights, and homepage featured cards now expose real destinations for native keyboard, copy-link, and new-tab behavior. Ordinary clicks keep in-app navigation; admin selection cards remain buttons. Highlight and hero page controls are sibling buttons, and destinations track the selected image.

Carousel drag suppression is cleared between gestures and never blocks keyboard activation. Retain active-only showcase image loading and the homepage's three-image window. Validation: all 1,136 frontend tests and the production build pass, including keyboard highlight controls and modifier-click behavior. Browser regressions cover middle-click opening and Back/Forward search history. Earlier combined validation passed all 83 mocked browser tests; current-base CI repeats those checks before merge.

### CODE REVIEW

File: frontend/src/components/LetterCard/LetterCard.tsx [hunk 1]
```diff
@@ -323,18 +323,21 @@ function LetterCard({
       onMouseEnter={handleCardMouseEnter}
       onMouseLeave={handleCardMouseLeave}
     >
-      <button
-        type="button"
+      <a
+        href={`/letter/${card.id}`}
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/components/LetterCard/LetterCard.tsx]: Added an explicit selection variant and applied it to both admin featured-item pickers. It renders a button without a destination and retains selection on modified clicks. Public catalogue instances remain native links. Regression tests and frontend build pass.

**Comment by MLGalusha** [comment]
@codex review

File: frontend/src/pages/HomePage.tsx [hunk 2]
```diff
@@ -175,15 +171,15 @@ function HeroLetterCard({
...
-      role="button"
-      tabIndex={0}
       className={`letter-card home-hero-feature-card letter-card--${heroLetter.imageType || 'letter'}`}
-      aria-label={ariaLabel}
-      onPointerDown={handlePointerDown}
-      onClick={navigateToLetter}
-      onKeyDown={handleCardKeyDown}
       onMouseEnter={handleHeroHover}
     >
+      <a className="home-hero-open-link"
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/pages/HomePage.tsx]: Featured links now explicitly opt into carousel dragging with native drag disabled; modifier clicks remain native. The homepage carousel suppresses click-through after dragging. Added a drag-versus-click regression; carousel/homepage tests and frontend build pass.

**Comment by MLGalusha** [comment]
@codex review

File: frontend/src/components/LetterCard/LetterCard.tsx [hunk 3]
```diff
@@ -323,18 +326,22 @@ function LetterCard({
       onMouseEnter={handleCardMouseEnter}
       onMouseLeave={handleCardMouseLeave}
     >
-      <button
-        type="button"
+      <CardControl
+        type={selection ? "button" : undefined}
+        href={selection ? undefined : `/letter/${card.id}`}
         className={`letter-card letter-card--${card.imageType}${hasSearchPreview ? " letter-card--has-search-match" : ""}${previewVisible ? " letter-card--search-preview-visible" : ""}`}
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/components/LetterCard/LetterCard.tsx:332]: Public a.letter-card elements now restore -webkit-touch-callout: default; admin picker buttons retain their existing touch behavior. Production build passes. Desktop browser native-link tests remain in CI; an actual iOS long-press check is not claimed from the Chromium suite.

**Comment by MLGalusha** [comment]
@codex review

File: frontend/src/pages/HomePage.tsx [hunk 4]
```diff
@@ -129,7 +128,11 @@ function HeroLetterCard({
   const handlePointerDown = (e: React.PointerEvent) => {
     pointerStartRef.current = { x: e.clientX, y: e.clientY };
   };
-  const navigateToLetter = (e?: ReactMouseEvent) => {
+  const letterParams = new URLSearchParams({ from: 'highlight' });
+  if (currentImage) letterParams.set('image', currentImage.id);
+  const navigateToLetter = (e: ReactMouseEvent<HTMLAnchorElement>) => {
+    if (e.button !== 0 || e.metaKey || e.ctrlKey || e.shiftKey || e.altKey) return;
+    e.preventDefault();
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/pages/HomePage.tsx:136]: Each click now clears pointer history, keyboard-generated clicks bypass coordinate comparisons, and pointer cancellation clears the ref. Added a regression with a prior pointer-down followed by a keyboard-style click that verifies the reader route opens. Homepage tests and production build pass.

**Comment by MLGalusha** [comment]
@codex review

**Comment by MLGalusha** [comment]
@codex review

**Comment by MLGalusha** [comment]
Integrated the merged homepage loading changes. The hero remains a native anchor with sibling controls, retains the three-image window and independent feature loading, and no longer restores the old hover preload handler. Eighteen focused component tests, both public archive browser regressions, and the production build pass.

**Comment by MLGalusha** [comment]
@codex review

File: frontend/src/pages/HomePage.tsx [hunk 5]
```diff
@@ -367,7 +366,7 @@ export default function HomePage() {
       />
       {isMobile && (heroLetter || !heroLoaded) ? (
         <section className="home-hero home-hero--carousel">
-          <InfiniteCarousel classPrefix="hero-carousel" pauseRef={carouselPauseRef}>
+          <InfiniteCarousel classPrefix="hero-carousel" pauseRef={carouselPauseRef} suppressClickAfterDrag>
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/pages/HomePage.tsx:369]: Cleared drag suppression when the pointer leaves the track and before every new mouse gesture, including ordinary links excluded from dragging. Keyboard activation also bypasses pointer suppression. Added regressions for leaving the track then clicking a CTA, keyboard activation after a drag, and retained same-gesture drag click suppression. Fifteen focused tests and the production build pass.

**Comment by MLGalusha** [comment]
@codex review

**Comment by MLGalusha** [comment]
@codex review

**Comment by MLGalusha** [comment]
The final combined integration check also passes all 83 mocked browser tests and the lint regression gate with zero increases. Collection highlights retain active-only image mounting and now expose the selected image in a native destination, with separate keyboard-operable previous/next buttons.

File: frontend/src/components/ShowcaseCard.tsx [hunk 6]
```diff
@@ -40,12 +40,20 @@ export default function ShowcaseCard({ items, onNavigate, onInteraction }: Showc
...
-      type="button"
-      className={`cd-highlight-card cd-highlight-card--${item.mediaType}`}
-      onClick={() => onNavigate(item.letterId, item.imageId)}
-    >
+    <div className={`cd-highlight-card cd-highlight-card--${item.mediaType}`}>
+      <a className="cd-highlight-open-link" data-carousel-drag draggable={false}
+        href={`/letter/${item.letterId}?${params.toString()}`}
+        aria-label={[item.label, item.peopleLine, item.date, item.hook].filter(Boolean).join(', ')}
+        onClick={(event) => {
+          if (event.button !== 0 || event.metaKey || event.ctrlKey || event.shiftKey || event.altKey) return;
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/components/ShowcaseCard.tsx]: Made the navigation handler optional: public instances with a handler render an anchor, while previews without one render non-link content. Removed the no-op handler from the admin collection preview. A regression verifies the preview has no link or href while its page controls still work; affected component/page tests and the production build pass.

**Comment by MLGalusha** [comment]
@codex review

File: frontend/src/components/InfiniteCarousel.tsx [hunk 7]
```diff
@@ -199,9 +202,14 @@ export default function InfiniteCarousel({
     if (!mouseDownRef.current) return;
     mouseDownRef.current = false;
     endDrag();
+    draggedRef.current = false;
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/components/InfiniteCarousel.tsx]: Retained the drag marker across mouseleave so a release after re-entry is still suppressed. A new mousedown clears it before interactive-target exclusions, and keyboard activation bypasses it, so the later-gesture fix remains intact. Added the leave/re-enter/release regression alongside the existing next-CTA and keyboard checks; focused tests and build pass.

File: frontend/src/components/ShowcaseCard.css [hunk 8]
```diff
@@ -235,3 +235,27 @@
     opacity: 1;
   }
 }
+
+.cd-highlight-open-link {
+  position: absolute;
+  inset: 0;
+  color: inherit;
+  border-radius: inherit;
```
[the remark this answers is not in the document]
Comment (MLGalusha) [reply frontend/src/components/ShowcaseCard.css:244]: Added explicit text-decoration: none to the highlight link, matching the existing archive and homepage card presentation. Production build passes.

**Comment by MLGalusha** [comment]
@codex review
